code/src/java/pcgen/io/PCGVer2Creator.java

```diff
@@ -2552,7 +2511,7 @@ private void appendWeightLine(StringBuilder buffer)
 		buffer.append(IOConstants.LINE_SEP);
 	}
 
-	private String chosenFeats(PCTemplate pct)
+	private String chosenFeats(CDOMObject pct)
```

Comment:
This is where I don't agree with a universal "be generic" rule.  It misleads users into thinking this is universal, when it's actually not.  This only works for Templates, and should be left specific as a signal to developers.